main/com.spotify/manifest.tt

```diff
@@ -271,6 +271,18 @@ class @com.spotify
   }]
   #[confirm=enum(auto)];
 
+  action add_song_to_playlist(in req song : Entity(com.spotify:song),
+			      in req playlist: Entity(com.spotify:playlist))
```

Comment:
Do we want a better question than the default "What playlist would you like?"

Reply:
I think "What playlist would you like?" works well enough. I can't really think of a question that would work better.